### DISCUSSION TITLE
Подскажите нубу 2

### DISCUSSION THREAD

**Discussion by bigmaniak** [Anonym-tsk/nfqws-keenetic#72] [Q&A]
[Вот тут](https://github.com/Anonym-tsk/nfqws-keenetic/discussions/21) было обсуждение. Новый вопрос
Где можно достать отдельно модуль Netfilter?? для архетиктуры mipsel. Он же на кинетики отдельно устанавливается
не стихает интерес запустить nfqws на моей машине...

**Comment by unameras** [comment]
День добрый. Вопрос-то актуален или уже решилось?
На самом деле полагаю проще всего обновить роутер на keenetic OS 4.2.1, установить компонент "Протокол IPv6", при установке он потребует установить "Модули ядра подсистемы Netfilter", удалить nfqws командой `opkg remove nfqws-keenetic` и установить заново по инструкции последнюю версию, работает шикарно на всех устройствах

**Comment by Neytrino-OnLine** [comment]
> на моей машине

Вы бы модель написали...

**Reply by unameras** [reply]
Fibertool FT-AIR-DUO-G с прошивкой Wive-NG. Я похоже выше поторопился с подсказками)

**Comment by PhotoSCH** [comment]
На стареньких **ZyXEL Keenetic Extra** на прошивке 2.16.D.12.0-11 и **ZyXEL Keenetic Giga II** на 2.16.D.12.0-11 изначально небыло **Netfilter**.
В наборе компонентов поставил галочку **Протокол IPv6**, перегрузился. Появились Модули ядра подсистемы Netfilter и Пакет расширения Xtables-addons для Netfilter.

**Reply by Hafizoff** [reply]
Здравствуйте!
Сорри за некропостинг, но есть вопрос по вашим роутерам. У меня giga kn-1012 , установил и настроил nfqws без проблем. Но это устройство свежее. И теперь хочу купить родителям какой-нибудь старенький роутер, однако, никак не могу найти, а какие должны быть характеристики аппаратной части роутера? Сколько минимум нужно оперативки и внутренней памяти для нормальной работы? Вот у вашей giga II всего 16 мб встройки и 256 оперы. Этого хватает? А если оперы будет 128, а встройки 32, к примеру, как на вашем  keenetic extra?

**Reply by Neytrino-OnLine** [reply]
> есть вопрос

Оно должно подняться на любых зикселях с USB-портами (кроме: 4G II и 4G III) для которых доступна прошивка 2.x. Но на зикселях нет (и не будет) DoT и DoH (можно настроить альтернативу), но я бы рассмотрел что-нить более актуальное (если у вас ещё нет (в наличии) соответствующего зикселя - какая разница что покупать (лишь бы цена устраивала))...
Лично у меня - поднялось на Zyxel Keenetic II и на Zyxel Keenetic Viva - отдал друзьям (лежали без дела)

**Reply by Hafizoff** [reply]
Благодарю за ответ. На условном Omni c 128 Mb оперы и 32 Mb встроенной то есть будет работать нормально? Что вообще из современных посоветуете за минимальный прайс?

**Reply by Neytrino-OnLine** [reply]
> Omni

(Если не забывать об отсутствии DoT и DoH) должен работать нормально (на 4pda найдёте его тему - там есть прошлогодняя прошивка 2.16)...

**Reply by Hafizoff** [reply]
Я имел ввиду современный вариант Omni kn-1410. Или как вариант с магазина купить здесь и сейчас Keenetic Launcher kn-1221. Что скажете?

**Reply by Neytrino-OnLine** [reply]
> Omni kn-1410

Да

> Keenetic Launcher kn-1221

Не поддерживает накопители (только модемы) - entware на него не поставить...

**Reply by Hafizoff** [reply]
Вас понял. Благодарю!
